dashboard/charts.py

```diff
@@ -118,3 +119,37 @@ def get_waffle_chart(data: pd.DataFrame):
                                         'judge_type_id': None})
 
     result = get_gender_donut_chart(filtered)
+
+
+def generate_word_cloud(summary_texts):
+    """Generates the word cloud itself with the 
+    correct design."""
+    background_color = '#0e1117'
+    combined_text = ' '.join(summary_texts)
+    word_cloud = WordCloud(width=800, height=400, background_color=background_color,
+                           stopwords=STOPWORDS, contour_width=0,
+                           max_font_size=80, min_font_size=10,
+                           relative_scaling=0.5, random_state=42).generate(combined_text)
+    return word_cloud
+
+
+def get_summary_texts_from_db(conn, case_no):
+    """Gets the summary from the transcripts and 
+    adds it to a list."""
+    summary_texts = []
+    try:
```

Comment:
Great try and except to catch potential error when we cannot connect to the database!